man/systemd.unit.xml

```diff
@@ -1658,6 +1658,29 @@
           </listitem>
         </varlistentry>
 
+        <varlistentry>
+          <term><varname>ConditionMemoryPressure=</varname></term>
+          <term><varname>ConditionCPUPressure=</varname></term>
+          <term><varname>ConditionIOPressure=</varname></term>
+
+          <listitem><para>Verify that the overall system (memory, CPU or IO) pressure is below or equal to a threshold.
+          This setting takes a colon-separated tuple as argument. The first value is the slice unit under which
+          the pressure will be checked, and it is optional: if not specified, the overall system pressure will be
+          measured, instead of a particular cgroup's.</para>
```

Comment:
Written like this, I would expect `:10%` to be valid syntax, however it seems that the correct syntax is just `10%`. In other words, the text as it stands says that the setting is always a tuple of value separated by `:` where the first value is optional, thus it is natural to infer that the separator is not optional.

What about something like:  "This setting takes a the threshold value as argument. It can be specified [insert text from second paragraph]. Optionally, the threshold value can be prefixed with the a slice unit under which the pressure will be checked, followed by a colon. If the slice unit is not specified, the overall system pressure will be measured, instead of a particular cgroup's."